apps/ledger-live-desktop/tests/specs/speculos/delegate.spec.ts

```diff
@@ -62,3 +92,38 @@ for (const account of accounts) {
     );
   });
 }
+
+for (const validator of validators) {
+  test.describe("Select a validator", () => {
+    test.use({
+      userdata: "skip-onboarding",
+      speculosApp: validator.delegate.account.currency.speculosApp,
+      cliCommands: [
+        `liveData --currency ${validator.delegate.account.currency.ticker} --index ${validator.delegate.account.index} --add`,
+      ],
+    });
+
+    test(
+      `[${validator.delegate.account.currency.name}] - Select validator`,
+      {
+        annotation: {
+          type: "TMS",
+          description: validator.xrayTicket,
+        },
+      },
+      async ({ app }) => {
+        await addTmsLink(getDescription(test.info().annotations).split(", "));
+        await app.layout.goToAccounts();
+        await app.accounts.navigateToAccountByName(validator.delegate.account.accountName);
+
+        await app.account.startStakingFlowFromMainStakeButton();
+        await app.modal.continue();
+
+        await app.delegate.verifyProvider(validator.delegate.provider);
+        await app.delegate.openSearchProviderModal();
+        await app.delegate.checkValidatorListIsVisible();
+        await app.delegate.selectProvider(1);
```

Comment:
you select provider, but you don't verify that it's actually displayed after closing search provider list